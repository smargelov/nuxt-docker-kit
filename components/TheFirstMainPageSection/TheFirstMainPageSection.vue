<template lang="pug">
  section.b-first-block
    h1._title TEST
      span._close
        SvgIcon(
          name="close"
          width="24"
          height="24"
        )._icon
</template>

<script>
export default {
  name: "TheFirstMainPageSection",
  props: {},
  data() {
    return {};
  },
  computed: {}
};
</script>

<style lang="sass">
@import './TheFirstMainPageSection'
</style>
