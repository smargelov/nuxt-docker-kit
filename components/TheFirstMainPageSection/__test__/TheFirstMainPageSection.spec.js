import { shallowMount } from "@vue/test-utils";
import TheFirstMainPageSection from "../TheFirstMainPageSection";

describe("TheFirstMainPageSection", () => {
  const wrapper = shallowMount(TheFirstMainPageSection);
  it("render correctly", () => {
    expect(wrapper.vm.$el).toMatchSnapshot();
  });
  test("is a Vue instance", () => {
    expect(wrapper.vm).toBeTruthy();
  });
});
