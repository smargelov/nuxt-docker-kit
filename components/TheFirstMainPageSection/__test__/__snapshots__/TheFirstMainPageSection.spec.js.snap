// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFirstMainPageSection render correctly 1`] = `
<section
  class="b-first-block"
>
  <h1
    class="_title"
  >
    TEST
    <span
      class="_close"
    >
      <svgicon
        class="_icon"
        height="24"
        name="close"
        width="24"
      />
    </span>
  </h1>
</section>
`;
